addons/mail/static/src/models/thread/thread.js

```diff
@@ -1245,6 +1258,18 @@ function factory(dependencies) {
         group_based_subscription: attr({
             default: false,
         }),
+        /**
+         * Determine whether this thread has the seen indicators (V and VV)
+         * enabled or not.
+         */
+        hasSeenIndicators: attr({
+            compute: '_computeHasSeenIndicators',
+            default: false,
+            dependencies: [
+                'mass_mailing',
+                'members',
+            ],
```

Comment:
outdated dependencies, should be `model` and `channel_type` instead of `members`